.github/workflows/conformance-datapath.yaml

```diff
@@ -0,0 +1,268 @@
+    needs: check_changes
+    name: Setup & Test
+    if: |
+      (github.event_name == 'issue_comment' && (
+        github.event.comment.body == '/ci-datapath' ||
+        (github.event.comment.body == '/test' && needs.check_changes.outputs.tested == 'true')
+      )) ||
+      github.event_name == 'schedule' ||
+      github.event_name == 'pull_request'
+    timeout-minutes: 45
+    steps:
+      - name: Set up job variables
+        id: vars
+        run: |
+          if [ ${{ github.event.issue.pull_request || github.event.pull_request }} ]; then
+            PR_API_JSON=$(curl \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+              ${{ github.event.issue.pull_request.url || github.event.pull_request.url }})
+            SHA=$(echo "$PR_API_JSON" | jq -r ".head.sha")
+          else
+            SHA=${{ github.sha }}
+          fi
+
+          echo ::set-output name=sha::${SHA}
+
+      - name: Set commit status to pending
+        uses: Sibz/github-status-action@650dd1a882a76dbbbc4576fb5974b8d22f29847f
+        with:
+          authToken: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.vars.outputs.sha }}
+          context: ${{ github.workflow }}
+          description: Cilium Datapath Conformance test in progress...
+          state: pending
+          target_url: ${{ env.check_url }}
+
+      - name: Checkout pull request for Helm chart
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          ref: ${{ steps.vars.outputs.sha }}
+          persist-credentials: false
+
+      - name: Fetch cilium-cli
+        id: cilium-cli
+        uses: shrink/actions-docker-extract@v1
+        with:
+          # This is hack. Under the hood, the "action-docker-extract" does
+          # "docker cp $(docker create $image):/$path ...". While the cli
+          # image doesn't have an entrypoint, the create cmd fails. So, in
+          # addition to the image, pass the "ls" which is used as the
+          # entrypoint. The hack will go away, once @brb will create a custom
+          # GH action based on https://git.sr.ht/~motiejus/undocker.
+          # Also, we rely on the ":latest" since the datapath conformance suite
+          # is heavily WIP.
+          image: 'quay.io/cilium/cilium-cli-ci:latest ls'
+          path: '/usr/local/bin/cilium'
+      - name: Install cilium-cli
+        shell: bash
+        run: |
+          mv ${{ steps.cilium-cli.outputs.destination }}/cilium ./cilium-cli
+
+      - name: Wait for images to be available
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Run tests
+        uses: brb/lvh-action@v1.5
```

Comment:
Similar to the question left regarding `quay.io/cilium/cilium-cli-ci:latest`. the same applies to the images in the lvh-action repository where it is using `quay.io/lvh-images/lvh:latest`.

Reply:
The `lvh` is also very WIP. So for now I'd stick to the latest version.